draft-ietf-rats-corim.md

```diff
@@ -775,45 +775,54 @@ configuration files, read-only memory, fuses, IO ring configuration, partial
 reconfiguration regions, etc. Measurements comprise raw values, digests, or
 status information.
 
-An environment has one or more measurable elements. Each element can have a
-dedicated measurement or multiple elements could be combined into a single
-measurement. Measurements can have class, instance or group scope.  This is
-typically determined by the triple's environment.
+An environment has one or more measured elements.
+A measured element can have a dedicated measurement, such as a security version number.
+Multiple elements could be combined into a single measurement, such as a running hash of multiple files.
+A measured element can also have several measurement values, such as a name, version, and digest.
+Measurements can have class, instance or group scope.
+A `measurement-map`'s scope is determined by the `environment-map` it is paired with.
 
 Class measurements apply generally to all the Attesters in the given class.
 Instance measurements apply to a specific Attester instance.  Environments
 identified by a class identifier have measurements that are common to the
 class. Environments identified by an instance identifier have measurements that
 are specific to that instance.
 
-The supply chain entity that is responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
-is by default the CoRIM signer. If a different entity is authorized to provide measurement values,
-the `authorized-by` statement can be supplied in the `measurement-map`.
-
+The supply chain entity that is defined as the authority responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
+is by default the CoRIM signer.
+If different authorities may provide measurement values, the `"__authorities"` statement can be supplied in the `measurement-map`.
+If the `"__authorities"` statement is used within a condition, at least one of the listed authorities must match the respective ACS entry for the statement to match.
 
 ~~~ cddl
 {::include cddl/measurement-map.cddl}
 ~~~
 
+
 The following describes each member of the `measurement-map`:
 
-* `mkey` (index 0): An optional unique identifier of the measured
-  (sub-)environment.  See {{sec-comid-mkey}}.
+* `"__authorities"`: The cryptographic identities of individuals or organizations that are
```

Comment:
Suggest moving this to the end and replacing it with something like:
```suggestion
The special map key NULL indicates that the corresponding value is a list of authorities for the measurement.
In an ACS entry this indicates which entity deckared the validity of the measurement.
In a reference value entry this provides a list of the authorities whose authority declaration is required.
```

Reply:
Oh interesting, I didn't think it was a list of required authorities but rather a list of allowed authorities. At least one must match. Perhaps authorities should be an explicit pattern that is consistent with that interpretation:

authority = $crypto-key-type-choice
authorities = [ + authority ] / #6.562([ + authorities ]) / #6.563([ + authorities ])

We ensure #6.563 has a subset relationship with the list it's matched against, so all listed authority patterns in #6.563 must match some entry of acs-authorities.

This is different from #6.562 since that is any pattern fully matches the value (not an entry of the value, presumed array)